e2e/paths/enterChildren_test.js

```diff
@@ -52,7 +52,7 @@ Scenario('completing entering child information in the c110a application', (I, e
   enterChildrenPage.enterSocialWorkerDetails();
   enterChildrenPage.defineChildAdditionalNeeds();
   enterChildrenPage.defineContactDetailsVisibility();
-  enterChildrenPage.defineAbilityToTakePartInProceedings();
+  enterChildrenPage.enterLitigationIssues('No', 'Mock reason');
```

Comment:
That does not make sense. When 'No' is passed as a first argument we should not need to pass second argument. Please remove as it is just misleading.

Reply:
Also sometimes we should say `Yes` so that we exercise ability to enter details of litigation issues.